tools/llvm-slicer-opts.cpp

```diff
@@ -378,6 +385,13 @@ SlicerOptions parseSlicerOptions(int argc, char *argv[], bool requireCrit,
     options.preservedFunctions = splitList(preservedFuns);
     options.removeSlicingCriteria = removeSlicingCriteria;
     options.forwardSlicing = forwardSlicing;
+    options.cutoffDiveging = cutoffDiverging;
+
+    if (options.cutoffDiveging && options.dgOptions.threads) {
+        llvm::errs() << "[llvm-slicer] threads are enabled, not cutting off "
```

Comment:
I agree that it is an overhead for the current code, but it would prepare the code for the future modifications and made clear the intended line of possible future modifications. Anyway, this is good for now.